tmva/tmva/src/MethodBDT.cxx

```diff
@@ -2837,6 +2881,9 @@ void TMVA::MethodBDT::MakeClassSpecificHeader(  std::ostream& fout, const TStrin
    TString nodeName = className;
    nodeName.ReplaceAll("Read","");
    nodeName.Append("Node");
+   fout << "#include <algorithm>" << std::endl;
+   fout << "#include <limits>" << std::endl;
+   fout << "   " << std::endl;
```

Comment:
Very minor. This should probably be just `fout << "" << std::endl;`

Reply:
This is actually copy paste old code, i fixed a few more, and i think fout << std::endl; is even better :)